code/modules/mob/living/carbon/xenomorph/abilities/queen/queen_powers.dm

```diff
@@ -373,13 +373,15 @@
 /datum/action/xeno_action/onclick/manage_hive/use_ability(atom/Atom)
 	var/mob/living/carbon/xenomorph/queen/queen_manager = owner
 	plasma_cost = 0
-	var/list/options = list("Banish (500)", "Re-Admit (100)", "De-evolve (500)", "Reward Jelly (500)", "Exchange larva for evolution (100)",)
+	var/list/options = list("Banish (500)", "Re-Admit (100)", "De-evolve (500)", "Reward Jelly (500)", "Exchange larva for evolution (100)")
 	if(queen_manager.hive.hivenumber == XENO_HIVE_CORRUPTED)
 		var/datum/hive_status/corrupted/hive = queen_manager.hive
 		options += "Add Personal Ally"
 		if(length(hive.personal_allies))
 			options += "Remove Personal Ally"
 			options += "Clear Personal Allies"
+	if(queen_manager.ovipositor)
+		options += "Toggle Egg Autoplanting"
```

Comment:
I don't really deem this something as "Managing the Hive" and is a toggle so ought to be better conveyed visually as one of the toggleable actions. I would prefer this to instead be `/datum/action/xeno_action/activable`. If you're concerned about the ease of hiding abilities, add a key modifier (e.g. Alt) you can hold and click on an action to immediately hide the clicked on action. It would then be revealed again later using: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/66d58e13-d0b9-42cb-9c85-40a5a163e54d)
But feel free to just add it as an action and we can make a different PR to make the show hide actions easier to use. (E.g. I would want it to make a to_chat message saying they hid whatever ability with a href they can click on to immediately call the `action_hide_menu` verb again to unhide something if they accidentally did it).